iree/base/CMakeLists.txt

```diff
@@ -455,10 +455,15 @@ iree_cc_test(
     iree::testing::gtest_main
 )
 
-iree_select_compiler_opts(_SYNCHRONIZATION_LINKOPTS
-  CLANG_OR_GCC
-    "-lpthread"
-)
+if(NOT ANDROID)
```

Comment:
Typically I'd use the positive case in the `if` statement if both cases need to be handled, so `if (ANDROID)` would be preferable to me. But a minor issue.

Reply:
I read "not android" as "normal" and "default", so it makes more sense to me to lead with that.